proxysql_node_monitor

```diff
@@ -98,10 +98,19 @@ fi
 # DEFAULTS
 if [[ -z $WRITE_HOSTGROUP_ID ]]; then
   WRITE_HOSTGROUP_ID="1"
+else
+  if [[ ! -z $WRITE_HG ]]; then
+    WRITE_HOSTGROUP_ID=$WRITE_HG
+  fi
```

Comment:
@rameshvs02 if the sourced file has no `HOSTGROUP_WRITER_ID` but we passed `--write-hg` as part of the scheduler call, it will assume `HOSTGROUP_WRITER_ID=1` instead of whatever we passed on `--write-hg`